packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -212,19 +253,31 @@ export class PreferencesEditorsContainer extends DockPanel {
     @inject(PreferenceProvider) @named(PreferenceScope.Workspace)
     protected readonly workspacePreferenceProvider: WorkspacePreferenceProvider;
 
-    private preferenceScope: PreferenceScope;
+    private _userPreferenceEditorWidget: PreferencesEditorWidget;
+    private _workspacePreferenceEditorWidget: PreferencesEditorWidget | undefined;
+    private _foldersPreferenceEditorWidget: PreferencesEditorWidget | undefined;
 
```

Comment:
For the tree private variables (_userPreferenceEditorWidget, _workspacePreferenceEditorWidget and _foldersPreferenceEditorWidget),
If we follow the Theia coding guide-lines, we should not use the "_" in front of the variables 
( https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names )